src/config/constants/colors.config.ts

```diff
@@ -128,72 +126,72 @@ export const DarkModeColorsConfig: PaletteOptions = {
   },
 };
 
-export const OceanBlueColorsConfig: PaletteOptions = {
+export const OceanBlueColorsConfig = {
   ...ColorsConfig,
   primary: {
-    main: '#0077BE',
-    light: '#4DA7E3',
+    main: '#0082A9',
+    light: '#00A6AD',
     dark: '#005C8E',
   },
 };
 
-export const OceanBlueDarkColorsConfig: PaletteOptions = {
+export const OceanBlueDarkColorsConfig = {
   ...DarkModeColorsConfig,
   primary: OceanBlueColorsConfig.primary,
 };
 
-export const MidnightPurpleColorsConfig: PaletteOptions = {
+export const MidnightPurpleColorsConfig = {
   ...ColorsConfig,
   primary: {
-    main: '#6A0DAD',
-    light: '#A14CC2',
+    main: '#7C36AA',
+    light: '#A85FD5',
     dark: '#520880',
   },
 };
 
-export const MidnightPurpleDarkColorsConfig: PaletteOptions = {
+export const MidnightPurpleDarkColorsConfig = {
   ...DarkModeColorsConfig,
   primary: MidnightPurpleColorsConfig.primary,
 };
 
-export const ForestGreenColorsConfig: PaletteOptions = {
+export const ForestGreenColorsConfig = {
   ...ColorsConfig,
   primary: {
-    main: '#228B22',
-    light: '#3CAE3A',
-    dark: '#006400',
+    main: '#00814C',
+    light: '#228B22',
+    dark: '#002D00',
   },
 };
 
-export const ForestGreenDarkColorsConfig: PaletteOptions = {
+export const ForestGreenDarkColorsConfig = {
   ...DarkModeColorsConfig,
   primary: ForestGreenColorsConfig.primary,
 };
 
-export const SunsetRedColorsConfig: PaletteOptions = {
+export const SunsetRedColorsConfig = {
   ...ColorsConfig,
   primary: {
-    main: '#FF4500',
-    light: '#FF7F50',
-    dark: '#CD3700',
+    main: '#D81900',
+    light: '#FF4500',
+    dark: '#8E0000',
   },
 };
 
-export const SunsetRedDarkColorsConfig: PaletteOptions = {
+export const SunsetRedDarkColorsConfig = {
   ...DarkModeColorsConfig,
   primary: SunsetRedColorsConfig.primary,
 };
 
-export const RoyalGoldColorsConfig: PaletteOptions = {
+export const RoyalGoldColorsConfig = {
   ...ColorsConfig,
   primary: {
-    main: '#FFD700',
-    light: '#FFEC8B',
-    dark: '#DAA520',
+    main: '#F2CC24',
+    light: '#E0BC00',
+    dark: '',
```

Comment:
this empty string for a colour throws an error. fix please 😽

Reply:
Done 🌼